packages/cli/src/index.ts

```diff
@@ -26,6 +26,9 @@ const main = async () => {
 
   const portalConfig = await cliConfig(args)
 
+  portalConfig.operatingSystemAndCpuArchitecture = args.arch
+  portalConfig.shortCommit = args.commit ?? (await simpleGit().revparse(['HEAD'])).substring(0, 7)
```

Comment:
I have to ask because I'm "that guy" but is it necessary to add a new dep for this or can we just execute a shell command for `git revparse` and assume that whoever is running this will have git installed locally?

Reply:
Changed it